docs/reference/hip_runtime_api_reference.rst

```diff
@@ -0,0 +1,54 @@
+.. meta::
+  :description: HIP runtime API reference page
+  :keywords: AMD, ROCm, HIP, CUDA, HIP runtime API, HIP runtime
+
+.. _runtime_api_reference:
+
+********************************************************************************
+HIP runtime API
+********************************************************************************
+
+This is the full HIP Runtime API reference. 
+
+Modules
```

Comment:
I would make this a reference to a separate page, and do the same for the Global defines, enums, structs and files, then have the list of features placed on the sub-modules. This would prevent the TOC from being the HIP Runtime API followed by a long list of sub-topics.

Reply:
Updated.